README.md

```diff
@@ -104,6 +104,7 @@ assert_eq!(total, dec!(27.26));
 * [rocket-traits](#rocket-traits)
 * [rust-fuzz](#rust-fuzz)
 * [std](#std)
+* [wasm](#wasm)
```

Comment:
This is no longer a feature but can be it's own section - something along the lines of Wasm support and not all features support wasm. Currently supported = xyz.